src/targets/gpu/include/migraphx/gpu/gemm.hpp

```diff
@@ -19,6 +21,7 @@ template <class Op>
 struct rocblas_gemm
 {
     Op op;
+    float beta          = 0;
```

Comment:
There should be an `alpha` field here as well.

Reply:
I can add it. but it won't be used. Only reason we have beta is that it allows fusion Gemm+Add fusion.

Reply:
alpha is added.